deploy/deploy.go

```diff
@@ -269,58 +288,51 @@ func doSingleUpload(ctx context.Context, bucket *blob.Bucket, upload *fileToUplo
 // localFile represents a local file from the source. Use newLocalFile to
 // construct one.
 type localFile struct {
-	// Path is the relative path to the file.
-	Path string
+	// NativePath is the native path to the file (using file.Separator).
```

Comment:
To make sure the code is clear, I replaced `Path` with `NativePath` (the local filesystem path, which may include backslashes for example) and `SlashPath`, the native path with `filepath.ToSlash` on it.

The code is now clear about using the `SlashPath` version when matching regular expressions for matchers or ordering, and also (critically) as the key in the map returned by `walkLocal`, so that we don't get tons of spurious diffs on Windows due to "/" vs "\\" mismatches.